Sources/Runtime/Config.swift

```diff
@@ -6,16 +6,33 @@ struct Config {
         let alwaysNotify: Bool?
         let openNewWindowsWith: String?
         let openNewWindowsInBackground: Bool?
+        let customCookie: [Config.Schema.Cookie]?
```

Comment:
I'd like to keep this config exclusively at the tab level.

Reply:
There's a reason I've added this on the global level as well.
The cookie injection is used in many instances where auto login is problematic (like Grafana) - and the logic token keep expiring.
If it has to be on the tab level, this means every time it expires - you'll have to manully update it for each tab and not once on the top level.

Reply:
In that scenario, I'd recommend using a separate tool (like a Bash script) to template/generate your JSON config. That's one of the advantages of having the config file live as a plain JSON file on disk.

Reply:
most of the users I've been installing the took on for the graphana. ... are not that advanced...
they just know to replace the token key in one location (they get it in the email)

I'll just keep my version for now - as we disagree on this when I prefer to have the ability to set global cookies (and allow to override them on a tab basis if needed)